tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MacroSuite.scala

```diff
@@ -421,15 +421,9 @@ class MacroSuite extends BaseDottySuite {
       )
     )
   }
+
   test("no-name") {
-    runTestAssert[Stat](
-      "'{ val x: Int = $ }"
-    )(
-      Term.QuotedMacroExpr(
-        Term.Block(
-          List(Defn.Val(Nil, List(Pat.Var(Term.Name("x"))), Some(Type.Name("Int")), Term.Name("$")))
-        )
-      )
-    )
+    runTestError[Stat]("'{ val x: Int = $ }", "error: Expected macro spliced ident")
```

Comment:
This can actually be correct single `$` can be a previously declared ident.

Reply:
should it be just `$`, or in reality `$$`?

how should this be encoded: `$+`, `$ + 1`?

Reply:
I don't think you can include + as part of the identifier.

It should be $, you can already do `val $ = 123` and as I checked with Scala 3 macros it accepts a single $

Reply:
done